packages/app/package.json

```diff
@@ -7,6 +7,9 @@
   "browser": "dist/esm/index.esm2017.js",
   "module": "dist/esm/index.esm2017.js",
   "esm5": "dist/esm/index.esm5.js",
+  "engines": {
```

Comment:
Since we're testing `typeof FinalizationRegistry`, I don't think we have to set an engine restriction—as doing so would constitute a major release IMO